iib/web/exception_handlers.py

```diff
@@ -0,0 +1,32 @@
+import functools
+
+import flask
+from kombu.exceptions import OperationalError
+
+from iib.exceptions import IIBError
+
+
+def catch_rabbit(request, messaging):
+    """Catches RabbitMQ error and raises IIBError instead."""
```

Comment:
Could you please update the doc string to reflect the parameters taken by this function?

Reply:
Yes, updated